src/repos/player.js

```diff
@@ -0,0 +1,148 @@
+var sql = require('pg-sql').sql
```

Comment:
Addendum that if  you want to switch out the existing SQL to use your preferred style (and can do so while keeping line lengths to 80 ideally) and it's still readable, I don't mind changing.